cli/cli/commands/cluster/set/set.go

```diff
@@ -60,51 +57,10 @@ func run(ctx context.Context, flags *flags.ParsedFlags, args *args.ParsedArgs) e
 		return nil
 	}
 
-	engineManagerOldCluster, err := engine_manager.NewEngineManager(ctx)
```

Comment:
note this creates a failure condition in case the local engine isn't stopped; where there would be portclash